src/ngraph/op/abs.hpp

```diff
@@ -29,18 +29,22 @@ namespace ngraph
         class Abs : public util::UnaryElementwiseArithmetic
         {
         public:
+            NGRAPH_API
+            static const std::string type_name;
+            const std::string& description() const override { return type_name; }
```

Comment:
You could put the static string (which should be `s_type_name`) inside this function body, containing everything a little more neatly.
```
{
    static string s_type_name{"Abs"};
    return s_type_name;
}
```
This would apply to all such instances

Reply:
I want to be able to say `Cos::type_name` in the future.